data/json/monsters/mammal.json

```diff
@@ -549,6 +557,7 @@
     "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
     "placate_triggers": [ "MEAT" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Fox has size "SMALL", why give it harvest animal_medium? It needs to be a new category for animal_small.

Also will need to remove the FUR/BONES flags like you've done similarly on line 422.

Reply:
Didn't look hard enough, foxes are smaller than I thought.